### PR TITLE
749 Fix button width

### LINKED ISSUE #749: Button size tweaks needed
## Fix
- [x] Refactor the way buttons are styled to make this easier
- [x] Change the styles to conform to the design below ...

## The bug
Buttons are not quite the same in size and appearance as the design files, and need slight changes to be closer to intended design. 

The main problem is the height of each buttons is 50px instead of 48px and the cause seems to mostly be the 1px border - I have suggested changes below that should help but may require interpretation to apply correctly in react etc. 

Below shows the current implementation (appears the same in chrome, firefox and safari (IE/edge not tested). Next to it is the screenshot from design files, and next to that my attempts to get as close as possible to the intended designs by editing CSS in browser developer tools. I am not sure if this helps achieve the intended results in the XPub font-end code.

I have also made suggestions to increase the minimum width to 120px, and increased the side padding to give buttons more emphasis. The font should also be set to Noto Sans Semibold if font rendering issues are resolved.

**Screenshots**
image

image

image

 
In addition to above:

- [x] The SUBMIT button should look the same as the NEXT and BACK buttons.
- [x] The NEW SUBMISSION button needs to be centered as the above and needs more padding to the sides.

### PR DESCRIPTION
**Pull request by SemraSh**
#### Background

What does this PR do?
Fixes the size of the padding from 12px to 18px on the Button

#### Any relevant tickets

Closes #749 

#### How has this been tested?
👀 
css change

#### UI Changes

- [x] Has been reviewed against Designs
- [x] Necessary updates to styleguide

### CODE REVIEW

File: client/elife-theme/src/elements/ui/Button.js [hunk 1]
```diff
@@ -60,8 +60,7 @@ const attention = css`
 export default css`
   font-size: ${th('fontSizeBaseSmall')}
   line-height: ${th('lineHeightBaseSmall')};
-  padding: ${th('space.2')};
-  padding-bottom: 9px;
+  padding: 12px 18px 9px 18px;
```
Comment (diversemix) [inline client/elife-theme/src/elements/ui/Button.js:63]: Should we express these in terms of grid units? @Cooryd ?

Comment (jyenes) [reply client/elife-theme/src/elements/ui/Button.js:63]: I think it's ok in that way @diversemix

**Review by jyenes — APPROVED** [review]

**Comment by hdrury1** [comment]
@chugginselifesciences and @SemraSh is it worth dealing with the semibold button labels in this PR too? I believe the text should be NotoSans font weight 400?

**Comment by SemraSh** [comment]
> @chugginselifesciences and @SemraSh is it worth dealing with the semibold button labels in this PR too? I believe the text should be NotoSans font weight 400?

@hdrury1 is there a relevant ticket for that ? Could you tag that please I can make the changes in this PR.

**Comment by chris-huggins** [comment]
> @chugginselifesciences and @SemraSh is it worth dealing with the semibold button labels in this PR too? I believe the text should be NotoSans font weight 400?

@hdrury1 Buttons have always been intended to be Noto Sans Semibold at 400. Browsers render them differently so it can look too heavy sometimes.  I believe `-webkit-font-smoothing: antialiased` would improve the look for enabled browsers.

Noto Sans (regular) at 400 would be too lightweight compared to original designs.

In chrome on Mac:

With Noto Sans 400 at 14px

image

With Noto Sans Semibold 400 at 14px 

image

With Noto Sans Semibold 400 at 14px with `-webkit-font-smoothing: antialiased`

image

Do you want to test across different browsers?

**Comment by chris-huggins** [comment]
@hdrury1 

Currently in Firefox on Mac:

image

And safari:

image

**Review by chris-huggins — APPROVED** [review]

**Comment by chris-huggins** [comment]
Some context... we historically had issues with inconsistent use of Noto Sans 700 and Noto Sans Semibold 400 to provide the semibold look. This essentially is the result of some weirdness with the typeface in that there is no 500 or 600 "semibold" weight for noto sans, it instead is treated as if it's a different typeface. At one point we ended up combining the 700 weight and the semibold font which ended up with what an ugly "super bold".

https://github.com/elifesciences/elife-xpub/issues/915

**Comment by SemraSh** [comment]
@chugginselifesciences confirmed the above issue is fixed. 
I am merging this PR.

**Pull request merged**